okta/policy.go

```diff
@@ -0,0 +1,150 @@
+package okta
+
+import (
+	"fmt"
+	"log"
+
+	articulateOkta "github.com/articulate/oktasdk-go/okta"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+// Basis of policy schema
+var basePolicySchema = map[string]*schema.Schema{
+	"name": &schema.Schema{
+		Type:        schema.TypeString,
+		Required:    true,
+		ForceNew:    true,
+		Description: "Policy Name",
+	},
+	"description": &schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Policy Description",
+	},
+	// Need to add an enhancement here, maybe we can provide a hierarchy system, priorityBefore = <refPolicy>
```

Comment:
Chatted with Steve, he had a good point on this, while it is hard to make this complete dynamic, we can have a hard failure when priority cannot be set (usually due to priority being set to a value higher then the number of resources). Implement that and remove this comment.